Source/GTMSessionFetcher.m

```diff
@@ -1916,7 +1916,7 @@ - (void)releaseCallbacks {
   self.retryBlock = nil;
   self.testBlock = nil;
   self.resumeDataBlock = nil;
-  if (@available(iOS 10.0, *)) {
```

Comment:
I believe we'd migrated to watchOS 6.0 as a minimum: https://github.com/google/gtm-session-fetcher/pull/239/commits

Reply:
We don't seem to have it in SwiftPM.  Land this and then update the Package.swift files to cover that afterwards and remove any availability checks?  Or swap to those fixes in this PR?

Reply:
Updated comment accordingly.